ddl/index_cop.go

```diff
@@ -151,11 +163,27 @@ func (c *copReqSender) run() {
 			p.chunkSender.AddTask(idxRs)
 		}
 		terror.Call(rs.Close)
-	}
+		return nil
+	})
+}
+
+func wrapInBeginRollback(se *sess.Session, f func(startTS uint64) error) error {
+	err := se.Begin()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	defer se.Rollback()
+	var startTS uint64
+	sessVars := se.GetSessionVars()
+	sessVars.TxnCtxMu.Lock()
+	startTS = sessVars.TxnCtx.StartTS
+	sessVars.TxnCtxMu.Unlock()
+	return f(startTS)
 }
 
 func newCopReqSenderPool(ctx context.Context, copCtx *copContext, store kv.Storage,
-	taskCh chan *reorgBackfillTask, checkpointMgr *ingest.CheckpointManager) *copReqSenderPool {
+	taskCh chan *reorgBackfillTask, sessPool *sess.Pool,
```

Comment:
Seems v6.5， there is no checkpointManager?

Reply:
It has nothing related to the checkpoint manager.